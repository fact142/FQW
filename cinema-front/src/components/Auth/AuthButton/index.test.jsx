import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import renderer from 'react-test-renderer';
import { AuthButton } from './index';


const buttonText = 'test text';

const TestingFormButton = () => <AuthButton text={buttonText} />;

describe('FormButton component', () => {
  it('should render correctly', () => {
    const tree = renderer
      .create(
        <TestingFormButton />,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
  it('should render text from props', () => {
    const { getByText } = render(
      <TestingFormButton />,
    );
    expect(getByText(buttonText)).toBeInTheDocument();
  });
});
