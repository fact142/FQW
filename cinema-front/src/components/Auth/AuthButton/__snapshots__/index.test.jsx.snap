// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormButton component should render correctly 1`] = `
<button
  className="button"
>
  test text
</button>
`;
